### PR TITLE
Cart Pickup Event Picker Changes 

### PR DESCRIPTION
**Pull request by WishingWell13**
# Info

Closes **#230**.

## Changes

- Removed the hover/outline thing, it has weird interaction with unlinked events
- Hid pickup event from user on cart page if a pickup event is full 
- before, you had to wait for checkout to fail to see if a pickup event is full
- Fixed styling for create pickup event button

# Type of Change

- [x] Bug Fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [x] on the live deployment preview on Desktop.
- [ ] on the live deployment preview on Mobile.
- [ ] I have added new Cypress tests that are passing.

# Checklist

- [x] I have performed a self-review of my own code.
- [x] I have followed the style guidelines of this project.
- [x] I have documented any new functions in `/src/lib/*` and commented hard to understand areas
      anywhere else.
- [x] My changes produce no new warnings.

# Screenshots

image

image

### Background Info
https://css-tricks.com/a-complete-guide-to-data-attributes/

### CODE REVIEW

File: src/components/events/EventCard/index.tsx [hunk 1]
```diff
@@ -74,7 +74,10 @@ const EventCard = ({
       <Link
         href={`${config.eventsRoute}/${uuid}`}
         data-community={community}
-        className={`${styles.container} ${borderless ? '' : styles.bordered} ${className || ''}`}
+        data-disabled={borderless ? 'true' : 'false'}
+        className={`${styles.container} ${borderless ? 'disabled' : styles.bordered} ${
+          className || ''
+        }`}
```
Comment (alexzhang1618) [inline src/components/events/EventCard/index.tsx]: does just `data-disabled={borderless}` work?
also seeking to learn: why `'disabled'` over `''` like for className?

File: src/pages/store/cart.tsx [hunk 2]
```diff
@@ -249,7 +249,11 @@ const getServerSidePropsFunc: GetServerSideProps = async ({ req, res }) => {
 
   const savedCartPromise = CartService.getCart({ req, res });
   const pickupEventsPromise = getFutureOrderPickupEvents(AUTH_TOKEN).then(events =>
-    events.filter(event => event.status !== 'CANCELLED')
+    events
+      .filter(event => event.status !== 'CANCELLED')
+      .filter(
+        event => (event.orders?.length ?? 0) <= (event?.orderLimit ?? Number.MAX_SAFE_INTEGER)
```
Comment (alexzhang1618) [inline src/pages/store/cart.tsx]: how about `!(event.orders && event.orderLimit && event.orders.length > event.orderLimit)`? Or maybe it makes more sense to write this out more verbosely so the logic is clear to readers

**Review by alexzhang1618 — APPROVED** [review]
Generally LGTM, just some nits

File: src/components/events/EventCard/index.tsx (hunk 1 above)
Comment (WishingWell13) [reply src/components/events/EventCard/index.tsx]: `''` works for class name, 

'data-disabled={borderless}' seems to work!

**Pull request merged**